website/pages/en/help.js

```diff
@@ -32,7 +32,7 @@ class Help extends React.Component {
       },
       {
         content:
-          'Many users idle on Discord in [#relay](https://discord.gg/0ZcbPKXt5bX40xsQ) on [Reactiflux](https://www.reactiflux.com/).',
+          'Many users idle on Discord of [Reactiflux](https://www.reactiflux.com/).',
```

Comment:
Think the correct grammer here needs to read: "Many users idle on Discord for Reactiflux.

```suggestion
          'Many users idle on Discord for [Reactiflux](https://www.reactiflux.com/).',
```